cmd/skopeo/generate_sigstore_key.go

```diff
@@ -0,0 +1,91 @@
+package main
+
+import (
+	"errors"
+	"fmt"
+	"io"
+	"io/fs"
+	"os"
+
+	"github.com/containers/image/v5/pkg/cli"
+	"github.com/containers/image/v5/signature/sigstore"
+	"github.com/spf13/cobra"
+)
+
+type generateSigstoreKeyOptions struct {
+	outputPrefix   string
+	passphraseFile string
+}
+
+func generateSigstoreKeyCmd() *cobra.Command {
+	var opts generateSigstoreKeyOptions
+	cmd := &cobra.Command{
+		Use:     "generate-sigstore-key --output-prefix PREFIX",
```

Comment:
I’m not sure about the `--output-prefix` approach — an alternative is to have `--output-public` and `--output--private`, which is more explicit, but also much more typing.

Reply:
I like the idea :+1: Naming is always hard and I do not have another idea.